saleor/settings.py

```diff
@@ -1059,3 +1059,6 @@ def SENTRY_INIT(dsn: str, sentry_opts: dict):
 # Library `google-i18n-address` use `AddressValidationMetadata` form Google to provide address validation rules.
 # Patch `i18n` module to allows to override the default address rules.
 i18n_rules_override()
+
+# number of webhooks to be processed in a single batch per app
+WEBHOOK_ASYNC_BATCH_SIZE = 100
```

Comment:
This is not going to be configured by env variables, so please move it to saleor/webhook/transport/asynchronous/transport.py